spec/menu_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'menu.rb'
+
+describe Menu do
+
+  subject(:menu) { described_class.new(food) }
+  let(:food) do
```

Comment:
Nice use of let method, to single out testing of the menu class so that it only relies on the test and non changing functions within the main program. Might be worth looking further into this with the use of a double, for example
`allow(food).to receive(:menu).and_return({ tofu_ramen: 3.99, veggie_curry: 2.99 })`